app/views/shared/_navbar.html.erb

```diff
@@ -1,12 +1,16 @@
 <nav class="header-nav">
-  <div class="container">
+  <div class="con">
```

Comment:
I recommend leaving that class as a container, remember to give css classes names more in line with their function. That will help you in the future to identify which classes you use and why.

Reply:
Done